backup/main.py

```diff
@@ -0,0 +1,65 @@
+import argparse
+import os
+import logging
+import sys
+from typing import Optional
+
+
+class Parser(argparse.ArgumentParser):
+    def error(self, message):
+        raise argparse.ArgumentError(None, message)
+
+
+class Backup:
+    token = str
+    output_dir = str
+    organization = str
+    repositories = Optional[list]
+
+    def __init__(self, token, output_dir, organization, repositories):
+        self.token = token
+        self.organization = organization
+        self.output_dir = output_dir
+        self.repositories = repositories
+        if not os.path.isdir(self.output_dir):
+            raise AttributeError('Output directory does not exist')
```

Comment:
Here is a good place to add logging. It could be warning with the directory creation as a next step.
Or another option raise critical error and stopping execution.
Also consider using `try with ... except`

Reply:
Fixed. It is warning with the directory creation as a next step now